### PR TITLE
CLOUDP-240677 Update SDK update process to handle minor versions and create jiras

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Update SDK update process to handle minor versions and create jiras
_Jira ticket:_ CLOUDP-240677

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by fmenezes** [comment]
this works and verified, created https://github.com/mongodb/mongodb-atlas-cli/pull/2826 and https://jira.mongodb.org/browse/CLOUDP-241394

File: .github/workflows/autoupdate-sdk.yaml [hunk 1]
```diff
@@ -13,23 +12,76 @@ jobs:
...
+      - id:   version
+        run:  echo "VERSION=$(curl -sSfL -X GET  https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name')" >> "$GITHUB_OUTPUT"
+      - run:  make update-atlas-sdk
       - name: Verify Changed files
         uses: tj-actions/verify-changed-files@1e517a7f5663673148ceb7c09c1900e5af48e7a1
         id: verify-changed-files
         with:
           files: |
              ./internal/**/*
+             go.mod
```
Comment (fmenezes) [inline .github/workflows/autoupdate-sdk.yaml:25]: this is the actual change to make it work for minor and patch versions

File: .github/dependabot.yml [hunk 2]
```diff
@@ -8,6 +8,8 @@ updates:
     open-pull-requests-limit: 20
     reviewers:
       - "mongodb/mongocli"
+    ignore:
+      - dependency-name: "go.mongodb.org/atlas-sdk/*"
```
Comment (fmenezes) [inline .github/dependabot.yml:12]: ignoring on dependabot

**Review by wtrocki — DISMISSED** [review]
LGTM

File: .github/dependabot.yml (hunk 2 above)
Comment (wtrocki) [reply .github/dependabot.yml:12]: Actualy I'm not sure that we should do this. Do we have other job covering minor updates?

**Review by wtrocki — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (fmenezes) [reply .github/dependabot.yml:12]: this one

**Pull request merged**